docker/util/run.bzl

```diff
@@ -479,4 +479,4 @@ commit_layer = struct(
 
 def _process_commands(command_list):
     # Use the $ to allow escape characters in string
-    return 'sh -c $\"{0}\"'.format(" && ".join(command_list))
+    return 'cmd $\"{0}\"'.format(" && ".join(command_list))
```

Comment:
@Irvenae Do you have suggestion how to use the correct shell based on the operating system?

Reply:
we could have process_commands_sh or process_commands_cmd
we select the correct one with
select({
        "//build_tools:windows": _process_commands_cmd,
        "//conditions:default": _process_commands_sh
    })

Have you tried replacing run with run_shell on L385 instead of modifying this line?
This seems to try to run the script. (probably under cmd currently)

Reply:
_process_command is used for expanding a template into a final script, which is then run in line L222 (which I changed).

Reply:
I was looking at a different rule (which works similarly :) )
In that case I think the first suggestion should work. I can look into that